README.md

```diff
@@ -55,9 +55,13 @@ Please [open issues](https://github.com/microsoft/containerregistry/issues) to p
 
     Yes, customers can browse the MAR discovery UI at [mcr.microsoft.com](https://mcr.microsoft.com/). Customers can view rich content like descriptions and tag listings for each artifact.
 
-* **Can I Onboard new Images to MAR?** 
+* **Can I onboard new images to MAR?** 
 
-    No, Given that MAR is meant to host only official Microsoft Container Images, only Internal Microsoft Product teams will be able to onboard images to the registry. Once onboarded, the images are available to all public users.
+    No, Given that MAR is meant to host only official Microsoft container images, only internal Microsoft product teams will be able to onboard images to the registry. Once onboarded, the images are available to all public users.
+
+* **Does MAR have an service level agreement (SLA)?**
+
+    MAR is globally distributed offering and does not have an SLA. You can view SLAs such as Azure Container Registry on [Microsoft SLA for Online Services](https://www.microsoft.com/licensing/docs/view/Service-Level-Agreements-SLA-for-Online-Services).

```

Comment:
Consider improving the wording here to say something like "view SLAs for products that MAR depends on like Azure Container Registry or Azure Front Door"